features/encrypt/core/src/main/java/org/apache/shardingsphere/encrypt/algorithm/like/CharDigestLikeEncryptAlgorithm.java

```diff
@@ -154,17 +154,13 @@ private String digest(final String plainValue) {
         StringBuilder result = new StringBuilder(plainValue.length());
         for (char each : plainValue.toCharArray()) {
             char maskedChar = getMaskedChar(each);
-            if ('%' == maskedChar) {
```

Comment:
If the result of the mask operation is % or _, should we keep the original character?

Reply:
The getMaskedChar function returns % or _ directly